styles/_global.scss

```diff
@@ -5,5 +5,12 @@
 }
 
 body {
+  margin: 0%;
```

Comment:
When you set something to 0, you don't have to specify a unit. There is already normalize.css in the project so that line is redundant anyway